app/javascript/components/StaffDashboard/index.js

```diff
@@ -53,38 +53,40 @@ class StaffDashboard extends React.Component {
     }
 
     return (
-      <div className={classes.dashboard}>
-        <Navbar></Navbar>
-        <div className={classes.content}>
-          <h1>Participant Dashboard</h1>
-          <div className={classes.tableContainer}>
-            <div>
-              <div className={classes.searchBar}>
-                <InputBase
-                  placeholder="filter participants"
-                  onChange={this.handleChange}
-                />
-                <IconButton type="submit" aria-label="search">
-                  <SearchIcon />
-                </IconButton>
+      <ThemeProvider theme={theme}>
+        <div className={classes.dashboard}>
+          <Navbar></Navbar>
+          <div className={classes.content}>
+            <h1>Participant Dashboard</h1>
+            <div className={classes.tableContainer}>
+              <div>
+                <div className={classes.searchBar}>
+                  <InputBase
+                    placeholder="filter participants"
+                    onChange={this.handleChange}
+                  />
+                  <IconButton type="submit" aria-label="search">
+                    <SearchIcon />
+                  </IconButton>
+                </div>
+                <table>
+                  <thead>
+                    <tr>
+                      <th>PARTICIPANT</th>
+                      <th>STATUS</th>
+                      <th>PAPERWORK</th>
+                      <th>CASE NOTES</th>
+                      <th>FORM STATUS</th>
+                      <th> </th>
```

Comment:
Should this be empty?

Reply:
Looks like I messed up the blame during the refactor but I think @didvi wrote this component. Is it good to take out? If so I can make a pull request to delete this and/or just delete it on this PR

Reply:
It was here because there is a column for the arrow icons, so the number of th matches the number of td elements. This will all be changed to MaterialUI components so let's leave it for now.